examples/jsm/offscreen/jank.js

```diff
@@ -42,4 +42,5 @@ function jank() {
 
 }
 
+export { initJank };
```

Comment:
What is the problem with default exports?

Reply:
This was meant for https://github.com/mrdoob/three.js/pull/26910. No problem, just needs to be handled similar to how Nodes does.